test/image_dart_compiling/transformer_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:transformer_test/utils.dart';
+import 'package:scissors/image_dart_transformer.dart';
+
+main() {
+  var iconSvg = r'''
```

Comment:
Let's agree the others could literally be improved? Correctness should trump consistency, although this is still a nit.